sentry-android-core/src/test/java/io/sentry/android/core/internal/util/AndroidRuntimeManagerTest.kt

```diff
@@ -64,6 +64,37 @@ class AndroidRuntimeManagerTest {
     StrictMode.setThreadPolicy(threadPolicy)
     StrictMode.setVmPolicy(vmPolicy)
 
+    // Run the function and assert LAX policies
+    try {
+      sut.runWithRelaxedPolicy<Unit> {
+        assertEquals(
+          StrictMode.ThreadPolicy.LAX.toString(),
+          StrictMode.getThreadPolicy().toString(),
+        )
+        assertEquals(StrictMode.VmPolicy.LAX.toString(), StrictMode.getVmPolicy().toString())
+        called = true
+        throw Exception("Test exception")
+      }
+    } catch (_: Exception) {}
```

Comment:
Maybe extend the test to assert that the exception is thrown, something like this:
```suggestion
    } catch (_: Exception) {
      exceptionThrown = true
    }
```